src/reducers/lobby.js

```diff
@@ -56,6 +58,20 @@ export default function lobbyReducer(state = initialState(), action) {
     // ok seems legit
     return merge(state, {game: gameInitialState(new Board())});
 
+  //shuffle teams and spymasters
+  case SHUFFLE_PLAYERS:
```

Comment:
might want to add a `if (game is in progress) return state;` to prevent accidental shuffling of teams during an in-progress game